File: Code/Back-end/src/main/java/com/example/harsha/service/DailyLogService.java
package com.example.harsha.service;

import java.util.List;

import com.example.harsha.entity.Dailylog;

public interface DailyLogService {
	public List<Dailylog> getAllDailyLogs();
	public String addDailyLog(Dailylog dailylog);
}

File: Code/Front-end/src/app/dailylog.service.ts
import { Injectable } from '@angular/core';
import {HttpClient} from '@angular/common/http';
import { from } from 'rxjs';
@Injectable({
  providedIn: 'root'
})
export class DailylogService {
  baseUrl="http://localhost:8080/";
  constructor(private httpClient:HttpClient) { }

  public addDailylog(dailylog){
    return this.httpClient.post(this.baseUrl+"addDailylog",dailylog,{responseType:"text" as "json"});
  }

  public getAllDailylogs(){
    return this.httpClient.get(this.baseUrl+"dailyLogs");
  }
}